contrib/exporters/secadvisor/mod/transform.go

```diff
@@ -200,12 +128,32 @@ func (ft *securityAdvisorFlowTransformer) getNetwork(f *flow.Flow) *SecurityAdvi
 		return nil
 	}
 
+	var err error
+	aName := ""
+	bName := ""
+
+	if f.NodeTID != "" {
+		if f.Network.A != "" {
+			aName, err = ft.resolver.IPToName(f.Network.A, f.NodeTID)
+			if err != nil {
+				aName = f.Network.A
+			}
+		}
+
+		if f.Network.B != "" {
+			bName, err = ft.resolver.IPToName(f.Network.B, f.NodeTID)
+			if err != nil {
+				bName = f.Network.B
```

Comment:
Suggestion: This logic can be refactored to another kind of Resolver: one that tries to resolve and if it gets an error then it simply returns the input IP address.

Reply:
ok